package.json

```
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "run-s lint:html lint:scss",
```

Looks like a copy-paste issue. There's no `lint:html` script so this will fail.

```suggestion
    "lint": "run-s lint:scss"
```